scripts/exportexcel.py

```
@@ -0,0 +1,98 @@
+
+
+def prettify_json(to_prettify):
+    return ', '.join([f"{key}={val}" for key, val in json.loads(to_prettify).items()])
+
+current_dir = Path(os.getcwd())
+parent_dir = Path(os.path.realpath(__file__)).parent.parent
+
+if current_dir != parent_dir:
+    raise Exception(f"Script can only be run in {parent_dir}")
+
+from src.jsonsync import JsonItem
```

Ruff (0.12.2)

Module level import not at top of file

(E402)